src/OpenLoco/Windows/Vehicle.cpp

```diff
@@ -2342,17 +2344,73 @@ namespace OpenLoco::Ui::Windows::Vehicle
             return Vehicles::OrderRingView(head->orderTableOffset);
         }
 
-        static std::pair<Map::Pos3, const char*> sub_470B76(const Vehicles::Order& order, uint8_t orderNum)
+        static std::pair<Map::Pos3, std::string> sub_470B76(const Vehicles::OrderRingView::Iterator& order, uint8_t orderNum)
         {
-            static loco_global<char[512], 0x0112CC04> _stringFormatBuffer;
+            std::stringstream ss;
+            ss << ControlCodes::inline_sprite_str;
+            // TODO: remove addition
```

Comment:
TODO

Reply:
i mean its still there. I could just go and address it though.

Reply:
If you intend to leave the comment that's fine, but I guess does it make sense to anyone else? Will they know what you mean if they come across it?